### PR TITLE
Revert changes to core SDK members that haven't been propagated to the g3 SDK yet

### PR DESCRIPTION
**Pull request by DaveShuckerow**
This is causing errors in a g3 roll.

### CODE REVIEW

File: packages/flutter/lib/src/painting/image_provider.dart [hunk 1]
```diff
@@ -486,7 +486,8 @@ class NetworkImage extends ImageProvider<NetworkImage> {
       request.headers.add(name, value);
     });
     final HttpClientResponse response = await request.close();
-    if (response.statusCode != HttpStatus.ok)
+    // ignore: deprecated_member_use
+    if (response.statusCode != HttpStatus.OK)
```
Comment (Hixie) [inline packages/flutter/lib/src/painting/image_provider.dart]: nit: prefer putting the ignores on the actual line that has the problem

**Comment by Hixie** [comment]
LGTM though it seems the vmservice.dart fix isn't quite right.

File: packages/flutter/lib/src/painting/image_provider.dart (hunk 1 above)
Comment (DaveShuckerow) [reply packages/flutter/lib/src/painting/image_provider.dart]: Done.

**Comment by DaveShuckerow** [comment]
We were able to bring g3's dart from dev.60 to dev.61.  I've changed this PR to make us compatible with that version.

**Pull request merged**